database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
```

Comment:
Does `minutes_item_file_id` refer to file id of an attached document? As I understand it, there can be many attached documents for a `minutes_item`.  Is there a reason for the inclusion of one file id here?

Reply:
Good catch, I didn't consider that. I think two options could be to store a list of `minutes_item_file_id` in `minutes_item`, or we can just rely on storing `minutes_item_id` in `minutes_item_file_id` to link them. Which do you guys think would be better?